modules/timepoint_list/php/timepoint_list.class.inc

```diff
@@ -44,14 +44,14 @@ class Timepoint_List extends \NDB_Menu
         // check user permissions
         if ($user->hasPermission('access_all_profiles')
             || (in_array(
-                $candidate->getData('CenterID'),
+                $candidate->getData('RegistrationCenterID'),
```

Comment:
not sure I agree with this one, I think we need to make it more intelligent and have it look for at lease one timepoint at the user's site.

also acceptable in phase 2

Reply:
copied a link to that comment in issue #4058 so that it gets tackled properly when coding phase 2. Thanks @ridz1208